spec.html

```diff
@@ -29998,7 +29998,7 @@ <h1>
             1. Let _body_ be ParseText(! StringToCodePoints(_bodyString_), _bodySym_).
             1. If _body_ is a List of errors, throw a *SyntaxError* exception.
             1. NOTE: The parameters and body are parsed separately to ensure that each is valid alone. For example, `new Function("/*", "*/ ) {")` is not legal.
-            1. NOTE: If this step is reached, _sourceText_ must match _exprSym_ (although the reverse implication does not hold). The purpose of the next two steps is to enforce any Early Error rules which apply to _exprSym_ directly.
+            1. NOTE: If this step is reached, _sourceText_ must conform to the grammar for _exprSym_ (although the reverse implication does not hold). The purpose of the next two steps is to enforce any Early Error rules which apply to _exprSym_ directly.
```

Comment:
```suggestion
            1. NOTE: If this step is reached, _sourceText_ must have the syntax of _exprSym_ (although the reverse implication does not hold). The purpose of the next two steps is to enforce any Early Error rules which apply to _exprSym_ directly.
```

We use this phrasing in `Function.prototype.toString`.

Reply:
Done! (Though I wonder if it should be `have the syntax of *an* _exprSym_`.)